public/styles/abstracts/_mixins.sass

```diff
@@ -33,7 +33,7 @@
   border: 0px solid variables.$background-secondary
   background-color: variables.$button-inactive
   font-family: variables.$family-subtitle
-  border-radius: 20px
+  border-radius: 9999px
```

Comment:
border-radius: 25%/50%; looks similar

Reply:
Percentages make it elliptical. While this is fine for some buttons, it becomes more noticeable for buttons that are longer, or not as tall.

| `25%/50%` | `9999px` |
|-------------|----------|
|  ![image](https://github.com/FAForever/website/assets/38820160/ec631040-3481-4cdd-9c16-698cc77fb644) | ![image](https://github.com/FAForever/website/assets/38820160/a1540694-d5c4-47bc-82e0-975c4a45155d) |


While the `9999px` looks weird, it's just a massive number so that you never have to worry about the button being too large. To be honest, `99px` probably would've been sufficient too 😅